README.md

```diff
@@ -2,11 +2,17 @@
 
 The goal of this repo is to host the work on the Data Food Consortium prototype.
 
+## Requirements
+
+The prototype expects an authorization server to run on http://localhost:8000.
+
+See [fake-auth-server-php](https://github.com/datafoodconsortium/fake-auth-server-php) to setup a fake authorization server using Docker.
+
 ## Develop
 
 The client app uses node modules and is bundled with [Parcel](https://parceljs.org/).
 
-```
+```sh
```

Comment:
@nclavaud : Is this sh call really useful ?

Reply:
Oh, that's not a shell call but github [code block syntax highlighting](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) :-)

Now the code block shows comments in light grey:
![screenshot from 2017-12-19 17-20-12](https://user-images.githubusercontent.com/1181770/34167151-33df61f4-e4e1-11e7-9bc8-3ce0b00a774c.png)

Could have been another commit to make it explicit.

Reply:
Didn't know about that actually.